atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -258,7 +258,7 @@ def group_manifests_and_tag(self, session, worker_digests):
             push_conf_registry.digests[image.tag] = digest
 
         self.log.info("%s: Manifest list digest is %s", session.registry, digest_str)
-        return digest
+        return manifest['repository'], digest
```

Comment:
`manifest` is the last item of `manifests` list, which gets created during one of the for-loops up above.
Can we make this a little bit less brittle? There could be a case where `manifest` is not even set at all!

Reply:
Correct, repo name should be fetched from the `ImageName` instance we use to fetch manifest digest.

Fixed